ngraph/python/tests/test_onnx/test_zoo_models.py

```diff
@@ -46,6 +46,18 @@
 
 MODELS_ROOT_DIR = tests.MODEL_ZOO_DIR
 
+def yolov3_post_processing(outputs):
```

Comment:
Let's move this function to the file containing the original test.

Reply:
As I understand tests are created in runtime, but probably it will be worth to consider move post-processing operations to separate file in the future